resources/public/api/conf/common/testing.md

```diff
@@ -6,32 +6,6 @@ If you have a specific testing need that is not supported in the sandbox, contac
 
 Some APIs may be marked \[test only\]. This means that they are not available for use in production and may change.
 
-### Dynamic
-
-Some endpoints support DYNAMIC gov test scenarios. The response is dynamic based on the request parameters:
-
-- Retrieve Other Income V2.0
-
 ### Stateful
 
-Some endpoints support STATEFUL gov test scenarios. Stateful scenarios work with groups of endpoints that represent
-particular types of submissions. For each type you can POST (or PUT) to submit or amend data, GET to retrieve or list
-data and DELETE to delete data. For example, with a STATEFUL gov test scenario a retrieval will return data based on
-what you submitted.
-
-The following groups are stateful in the sandbox:
-
-- Employment Resources V2.0
-- Non-PAYE Employment Income V2.0
-- Other Employment Income V2.0
-- Dividends Income V2.0
-- UK Dividends Income V2.0
-- Foreign Income V2.0
-- Employments Financial Details V2.0
-- Insurance Policies Income V2.0
-- Pensions Income V2.0
-- UK Savings Account V2.0
-- Capital Gains on Residential Property Disposals V2.0
-- Other Income V2.0
-- Savings Income V2.0
-- Other Capital Gains and Disposals V2.0
\ No newline at end of file
+All endpoints in this API support STATEFUL gov test scenarios. Stateful scenarios work with groups of endpoints that represent particular types of submissions. For each type you can POST (or PUT) to submit or amend data, GET to retrieve or list data and DELETE to delete data. For example, with a STATEFUL gov test scenario a retrieval will return data based on what you submitted.
```

Comment:
Would it be worth adding a note something like the following?
> The **Retrieve UK Savings Account Annual Summary**
>  and **Create and Amend a UK Savings Account Annual Summary** endpoints rely on calling the **Add a UK Savings Account** to obtain a valid savingsAccountId. Otherwise an MATCHING_RESOURCE_NOT_FOUND will be returned from both of these endpoints.

Or maybe this info should be added against STATEFUL in the GTS table for the endpoints???